src/dynamo0p3.py

```diff
@@ -2906,6 +2866,49 @@ def bc_kernel(self, function_space):
                                  "%vspace%get_boundary_dofs()"),
                        position=["before", position])
 
+    def operator_bcs_kernel(self, function_space):
+        ''' Supply necessary additional arguments for the kernel that
+        applies boundary conditions to a LMA operator '''
+        from f2pygen import DeclGen, AssignGen
+        self._arglist.append("boundary_dofs")
+        parent = self._parent
+        parent.add(DeclGen(parent, datatype="integer",
+                           pointer=True, entity_decls=[
+                               "boundary_dofs(:,:) => null()"]))
+        # Find the single, LMA operator argument that this kernel updates
+        # TODO replace this with args_filter() once the modification to
+        # make it a module function is on master
```

Comment:
This is now redundant since you pointed out that we should only have 1 LMA operator argument anyway. I've put in sanity checks to ensure that's what we have.